smart_tests/commands/test_path_writer.py

```diff
@@ -20,7 +21,9 @@ def __init__(self, app: Application):
     def default_formatter(cls, x: TestPath):
         """default formatter that's in line with to_test_path(str)"""
         file_name = x[0]['name']
-        if cls.base_path:
+        # Only prepend base_path if it was explicitly set via --base option
+        # Auto-inferred base paths should not affect output formatting
+        if cls.base_path and cls.base_path_explicitly_set:
```

Comment:
I'm not sure what has changed compared to before.

It seems the caller set base_path
```py
    TestPathWriter.base_path = base_path
    TestPathWriter.base_path_explicitly_set = (base_path is not None)
```

Reply:
Also, I cannot find the place the base_path is set automatically.

Could you tell me the place??

Reply:
The place is the changes in `dynamic_commands.py`. But I know it's not intuitive. I'm going to revise this PR after my vacation.